src/ast/parameter.js

```diff
@@ -8,6 +8,13 @@
 const Declaration = require("./declaration");
 const KIND = "parameter";
 
+// eslint-disable-next-line no-unused-vars
+const MODIFIER_PUBLIC = 1;
```

Comment:
@czosel not sure thats a good approach, flags are better in memory usage but not readable, here I've used a different approch in other classes : https://github.com/glayzzle/php-parser/blob/master/src/ast/classconstant.js#L12